content/docs/tasks/security/mutual-tls/index.md

```diff
@@ -69,11 +69,19 @@ Please check [Istio identity](/docs/concepts/security/#istio-identity) for more
 
 ## Verify mutual TLS configuration
 
-You can use the `istioctl` tool to check the effective mutual TLS settings. To identify the authentication policy and destination rules used for the
-`httpbin.default.svc.cluster.local` configuration and the mode employed, use the following command:
+For this task, you may want to get the pod ID of the client app (`sleep`) with the following command:
 
 {{< text bash >}}
-$ istioctl authn tls-check httpbin.default.svc.cluster.local
+$ SLEEP_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name})
+{{< /text >}}
+
+You can use the `istioctl` tool to check the effective mutual TLS settings. The command requires the client pod as the destination rule depends on the client namespace.
```

Comment:
Should readers do these things now? Later? What command are you talking about, the one on line 84?

Reply:
Yes.

Reply:
```suggestion
Use the `istioctl` tool to check if the mutual TLS settings are in effect. The `istioctl` command needs the client's pod because the destination rule depends on the client's namespace.
```